### DISCUSSION TITLE
User custom role page not found

### DISCUSSION THREAD

**Discussion by NecoHorne** [directus/directus#5192] [Archive]
Hi 

I am new to Directus, I checked the docs and can't find an answer for this. 

When I create a new non admin Role, the user's assigned under that role cannot access their own user details. The response is Page not Found. I have given the role full read access to the Directus Users property with no luck. 

I have also tried adding the raw modules as specified [here](https://docs.directus.io/guides/roles/), also with no luck.

Has anybody found a similar issue or know of a fix to this? Thought I would ask in discussion before opening an issue. 

### Project details
Directus Version: 9.0.0-rc.59
Environment: production
OS: Linux 5.4.0-70-generic
Node: 10.19.0

**Comment by rijkvanzanten** [comment]
Hi @NecoHorne! This is a bug (https://github.com/directus/directus/issues/5130) which was fixed in https://github.com/directus/directus/commit/6b71398fc5ccb1b13ed80b1f35f9b43f1859927a and will be released later today in rc.60 👍🏻

**Reply by NecoHorne** [reply]
Awesome thank you so much, will wait for the update!
